vertical-pod-autoscaler/deploy/k8s-metrics-server.yaml

```diff
@@ -0,0 +1,244 @@
+---
+# Source: metrics-server/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: release-name-metrics-server
+  namespace: default
+  labels:
+    helm.sh/chart: metrics-server-3.8.3
+    app.kubernetes.io/name: metrics-server
+    app.kubernetes.io/instance: release-name
+    app.kubernetes.io/version: "0.6.2"
+    app.kubernetes.io/managed-by: Helm
+---
+# Source: metrics-server/templates/clusterrole-aggregated-reader.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:metrics-server-aggregated-reader
+  labels:
+    helm.sh/chart: metrics-server-3.8.3
+    app.kubernetes.io/name: metrics-server
+    app.kubernetes.io/instance: release-name
+    app.kubernetes.io/version: "0.6.2"
+    app.kubernetes.io/managed-by: Helm
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+rules:
+  - apiGroups:
+      - metrics.k8s.io
+    resources:
+      - pods
+      - nodes
+    verbs:
+      - get
+      - list
+      - watch
+---
+# Source: metrics-server/templates/clusterrole.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:release-name-metrics-server
```

Comment:
Is this role name one that SIG Architecture have confirmed? See https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-subjects for details about reserved names.

Reply:
Ah, this one's a mistake.  I did a simple `helm render` of the original source https://github.com/kubernetes-sigs/metrics-server/blob/master/charts/metrics-server/templates/clusterrole.yaml#L5 but didn't give it a decent name.  I'll update (I'll have a look at the URI you've provided)